src/app/messages/en/modalErrorText.ts

```diff
@@ -1,4 +1,4 @@
 export const bodyWithLink = `
-  We're not quite sure what went wrong. Refresh the page. If this doesn't solve
+  We're not quite sure what went wrong. Restart your browser. If this doesn't solve
```

Comment:
For Spanish, replace "Actualice la página." with "Reinicia el navegador."

Reply:
Thanks!